src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -32,6 +33,17 @@ export const authOptions: NextAuthOptions = {
           scope: GOOGLE_API_SCOPE.join(" "),
         },
       },
+      profile: (profile) => {
```

Comment:
`profile` is populated from GoogleProvider. The interface for `profile` is defined [here](https://github.com/nextauthjs/next-auth/blob/v4/packages/next-auth/src/providers/google.ts#L3-L18)